composables/use-text-to-speech.ts

```diff
@@ -246,33 +249,44 @@ export function useTextToSpeech(options: TTSOptions = {}) {
     })
     audio.src = `/api/reader/tts?${params.toString()}`
     audio.playbackRate = ttsPlaybackRate.value
+    audio.setAttribute('data-text', element.text)
 
     return audio
   }
 
-  function playNextElement({ reset = false } = {}) {
-    if (currentTTSSegmentIndex.value + 1 >= ttsSegments.value.length) {
-      return
-    }
-    currentTTSSegmentIndex.value += 1
+  function playCurrentElement(delay = 0) {
```

Comment:
Maybe use object params?
```suggestion
  function playCurrentElement({ delay = 0 } = {}) {
```